Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -152,11 +259,60 @@ private void handlePayPalResult(PaymentMethodNonce paymentMethodNonce) {
             super.onPaymentMethodNonceCreated(paymentMethodNonce);
 
             PayPalFragmentDirections.ActionPayPalFragmentToDisplayNonceFragment action =
-                PayPalFragmentDirections.actionPayPalFragmentToDisplayNonceFragment(paymentMethodNonce);
+                    PayPalFragmentDirections.actionPayPalFragmentToDisplayNonceFragment(paymentMethodNonce);
             action.setTransactionAmount(amount);
             action.setDeviceData(deviceData);
 
             NavHostFragment.findNavController(this).navigate(action);
         }
     }
+
+    @OptIn(markerClass = ExperimentalBetaApi.class)
+    private void launchPayPalEditFIVault(String editVaultId) {
```

Comment:
Can we combine this and the below method since the logic is the same mostly? We should be able to do something similar to what we did on iOS here: https://github.com/braintree/braintree_ios/blob/88c99c12ea7ef9d3b40f8242c88ef50f8aab9030/Demo/Application/Features/PayPalWebCheckoutViewController.swift#L227-L234